src/main/java/Invokers/ApiClient.java

```diff
@@ -1067,11 +1067,7 @@ public File prepareDownloadFile(Response response) throws IOException {
 						|| headerType.equals("application/pdf")) {
 					responseBody = response.body().string();
 				}
-			if (!(responseCode.equals("200"))) {
-				if (!responseCode.equals("201")) {
-					System.out.println(response.body().string());
-				}
-			}
+
```

Comment:
It may solve the exception here, but I guess it sill still happen later on, at https://github.com/CyberSource/cybersource-rest-client-java/blob/master/src/main/java/Invokers/ApiClient.java#L1174